Explorer/Assets/Scripts/Utility/Primitives/PlaneFactory.cs

```diff
@@ -0,0 +1,83 @@
+using UnityEngine;
+
+namespace Utility.Primitives
+{
+    public static class PlaneFactory
+    {
+        public const int VERTICES_NUM = 8;
+        public const int TRIS_NUM = 12;
+        public static readonly Vector3 SIZE = PrimitivesSize.PLANE_SIZE;
```

Comment:
It does not make sense to keep an additional constant, just use `PrimitivesSize` directly